java/src/org/openqa/selenium/manager/SeleniumManager.java

```diff
@@ -111,26 +111,28 @@ private static String runCommand(String... command) {
           CharStreams.toString(
               new InputStreamReader(process.getInputStream(), StandardCharsets.UTF_8));
     } catch (InterruptedException e) {
-      LOG.warning(
-          String.format(
-              "Interrupted exception running command %s: %s",
-              Arrays.toString(command), e.getMessage()));
       Thread.currentThread().interrupt();
+      throw new WebDriverException(
+        "Interrupted while running command: "
+          + Arrays.toString(command),
+          e);
     } catch (Exception e) {
-      LOG.warning(
-          String.format(
-              "%s running command %s: %s",
-              e.getClass().getSimpleName(), Arrays.toString(command), e.getMessage()));
+      throw new WebDriverException(
+        "Failed to run command: "
+          + Arrays.toString(command),
+          e);
     }
-    SeleniumManagerJsonOutput jsonOutput =
-        new Json().toType(output, SeleniumManagerJsonOutput.class);
-    if (code > 0) {
```

Comment:
Selenium Manager can return a code different than 0 but it will still return the valid error in a JSON payload. The new code will output the whole payload which is not necessary.

Reply:
The new version will now try to parse the response in any way.
I handled the empty string to avoid the misleading exception we hand in other cases.

Reply:
Thank you